payment/src/main/java/com/kyriba/conference/payment/domain/PaymentMethodEntity.java

```diff
@@ -0,0 +1,44 @@
+package com.kyriba.conference.payment.domain;
+
+import com.kyriba.conference.payment.api.dto.PaymentMethodDto;
+import com.kyriba.conference.payment.api.dto.PaymentMethodResponse;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "payment_method")
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class PaymentMethodEntity {
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "seq_identity_id")
+    @SequenceGenerator(name = "seq_identity_id", sequenceName = "seq_identity_id", allocationSize = 1)
+    @Id
+    @Column(name = "id", updatable = false, nullable = false)
```

Comment:
The same comments as for another entity.

Reply:
fixed